sdk/keyvault/azure-security-keyvault-common/CHANGELOG.md

```diff
@@ -2,3 +2,6 @@
 
 ## 4.0.0-beta.1 (Unreleased)
 
+### New Features
+
+- KeyVaultException.
```

Comment:
Might want to be more descriptive. This should be a descriptive description of what changed. Changelogs should also use headers Added, Changed, and Removed last I saw in our SDK recommendations (might want to double check).